tds/src/main/java/thredds/server/wfs/WFSController.java

```diff
@@ -23,7 +23,27 @@
 @RequestMapping("/wfs")
 public class WFSController extends HttpServlet {
 
-	public static final String TDSNAMESPACE = "\"http://localhost:8080/thredds/wfs/results\"";
+	public static final String TDSNAMESPACE = "tdswfs";
+	
+	/**
+	 * Gets the namespace associated with the WFS Controller and this specific THREDDS server
+	 * 
+	 * @param req The request which contains the applicable URI of the WFS Controller
+	 * @return the namespace value as "SERVER/PATH"
+	 */
+	public static String getXMLNamespaceXMLNSValue(HttpServletRequest hsreq) {
+		return hsreq.getScheme() + "://" + hsreq.getServerName() + ":" + hsreq.getServerPort() + "/thredds/wfs/geospatial";

```

Comment:
Should this delegate to `constructServerPath()`?

Reply:
Oh yeah, sure, will change it to partially delgate to the server path. This was a result of copying and pasting...!